pkg/runtime/runtime.go

```diff
@@ -1295,11 +1295,11 @@ func (a *DaprRuntime) createAppChannel() error {
 func (a *DaprRuntime) announceSelf() error {
 	switch a.runtimeConfig.Mode {
 	case modes.StandaloneMode:
-		err := discovery.RegisterMDNS(a.runtimeConfig.ID, a.runtimeConfig.InternalGRPCPort)
+		err := discovery.RegisterMDNS(a.runtimeConfig.ID, []string{a.hostAddress}, a.runtimeConfig.InternalGRPCPort)
 		if err != nil {
 			return err
 		}
-		log.Info("local service entry announced")
+		log.Infof("local service entry announced: %s -> %s:%d", a.runtimeConfig.ID, a.hostAddress, a.runtimeConfig.InternalGRPCPort)
```

Comment:
log.Info (remove extra f)

Reply:
this will remove the format - this prints the service name and the service entry address and port. Without that it's opaque.

Reply:
Infof makes sense to log more information.